### PR TITLE
Updating core documentation

### PR DESCRIPTION
**Pull request by david-waltermire-nist**
# Committer Notes

Updating readmes with more current and relevant information.
Adding CODEOWNERS to drive reviews.

### All Submissions:

- [x] Have you selected the correct base branch per  [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) guidance?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]
Looks good. Found a minor pre-existing space issue, but I learned some new things today. :-)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+/build/                 @usnistgov/itl-oscal-maintainers
+/content/               @david-waltermire-nist
+/docs/                  @david-waltermire-nist
+/docs/content           @david-waltermire-nist @iMichaela
+/json/                  
+/xml/                   
+/src/                   @usnistgov/itl-oscal-maintainers
+/src/metaschema         @david-waltermire-nist
+/src/specifications/profile-resolution  @stephenbanghart
+/src/utils              @wendellpiez
```
[the remark this answers is not in the document]
Comment (david-waltermire-nist) [reply .github/CODEOWNERS:12]: Interesting. The [code owners documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) states:

> The people you choose as code owners must have read permissions for the repository. When the code owner is a team, that team must be visible and it must have write permissions, even if all the individual members of the team already have write permissions directly, through organization membership, or through another team membership.

[the remark this answers is not in the document]
Comment (aj-stein-nist) [reply .github/CODEOWNERS:12]: Interesting, so we're good? If it works that's fine. I had not checked if the team is public/private (I was aware of the toggle, [this maintainers team is still secret](https://github.com/orgs/usnistgov/teams/itl-oscal-maintainers)), but then again, that is not where the validation highlights occur, so yeah ... haha.

[the remark this answers is not in the document]
Comment (david-waltermire-nist) [reply .github/CODEOWNERS:12]: I think we should try it.

**Review by aj-stein-nist — APPROVED** [review]
Per discussion with Dave, the only concern I had was the GitHub warnings on the CODEOWNERS file. Everything else looks good, so we can just move forward and test it.

**Pull request merged**